packages/aws-cdk-lib/aws-eks/lib/managed-nodegroup.ts

```diff
@@ -282,6 +282,25 @@ export interface NodegroupOptions {
    * @default - ON_DEMAND
    */
   readonly capacityType?: CapacityType;
+  /**
+   * The maximum number of nodes unavailable at once during a version update.
+   *
+   * Nodes will be updated in parallel. This value or `maxUnavailablePercentage` is required to have a value. The maximum number is 100.
+   *
+   * @see https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/aws-properties-eks-nodegroup-updateconfig.html#cfn-eks-nodegroup-updateconfig-maxunavailable
+   * @default 1
+   */
```

Comment:
I put this at 1 because if left undefined we actually still apply an updateConfig maxUnavailable value of 1.  I could set it as undefined, but I think that's incongruent with what's actually happening behind the scenes.

Reply:
You're right. Thank you for clarifying 👍